x-pack/plugins/canvas/public/state/initial_state.js

```diff
@@ -22,11 +22,11 @@ export const getInitialState = path => {
       // values in resolvedArgs should live under a unique index so they can be looked up.
       // The ID of the element is a great example.
       // In there will live an object with a status (string), value (any), and error (Error) property.
-      // If the state is 'error', the error proprty will be the error object, the value will not change
+      // If the state is 'error', the error property will be the error object, the value will not change
       // See the resolved_args reducer for more information.
     },
     persistent: {
-      schemaVersion: 1,
+      schemaVersion: 2,
```

Comment:
It didn't look like we _neeed to_ increment, I just acted on [an earlier feedback item](https://github.com/elastic/kibana/pull/25854#issuecomment-440407769), though that might have been under different assumptions. 

We don't need to migrate, so this can be reverted; on the other hand, the changes do represent a schema change (new props) so I didn't question the original feedback item, looked totally sensible.

In short, it can go either way; no need for migration.

Reply:
OK, might as well bump it then.